apps/web/pages/api/book/instant-event.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+import handleInstantMeeting from "@calcom/features/instant-meeting/handleInstantMeeting";
+import { checkRateLimitAndThrowError } from "@calcom/lib/checkRateLimitAndThrowError";
+import getIP from "@calcom/lib/getIP";
+import { defaultResponder } from "@calcom/lib/server";
+
+async function handler(req: NextApiRequest & { userId?: number }, res: NextApiResponse) {
+  const userIp = getIP(req);
+
+  await checkRateLimitAndThrowError({
+    rateLimitingType: "core",
+    identifier: userIp,
```

Comment:
Lets make this more specific so we can get a detailed breakdown in upstash of where things could be potentially abused
```suggestion
    identifier: `instant.event-${userIp}`,
```